themes/brainy/brainy.theme.bash

```diff
@@ -208,16 +208,16 @@ _brainy_completion() {
 	segments="battery clock exitcode python ruby scm sudo todo"
 	case "${_action}" in
 		show)
-			COMPREPLY=( $(compgen -W "${segments}" -- "${cur}") )
+			read -r -a COMPREPLY <<< "$(compgen -W "${segments}" -- "${cur}")"
```

Comment:
this does not work as expected, it only reads the first word. I opened #1791 to disable this warning

Reply:
Thanks for finding this issue.
After some tests, I found a solution: https://github.com/Bash-it/bash-it/pull/1791#issuecomment-761688671